### ISSUE TITLE
file is in html and not an expected binary file

### ISSUE THREAD

**Issue by daniellukeahead** [olivierhagolle/LANDSAT-Download#54]
Hi @olivierhagolle  ,

I have used this code quite well for several months, but I just got a new issue with the follows:

![Capture](https://user-images.githubusercontent.com/49530841/55976277-02bfc100-5c49-11e9-8015-6fa0d1ab2b52.PNG)

The input of the URL(_'https://earthexplorer.usgs.gov/download/12864/LC80420202015242LGN01/STANDARD/EE'_) in google chrome directly come up with the download. but the code is just not running through it. 

Any suggestion to fix this problem please?

Thank you!

**Comment by olivierhagolle** [comment]
Hi, did you have a look to the resulting html file ./error_output.
It should contain the cause of the error.
Olivier

**Comment by daniellukeahead** [comment]
Hi, Thanks for your reply. 
Yes, I did. As it is shown below, I did not get any relevant information. I can log in with the username and password. According to the debugger, the file retrieved by the urllib2.urlopen(url) is in html instead of a binary file. That might be the cause of the error, but I have no idear how to fix it. 
![Capture](https://user-images.githubusercontent.com/49530841/56082894-94f7ce80-5ddb-11e9-8406-0b0b1369fb39.PNG)

I was wondering if the USGS had made any modification somehow because the code used to work very well if the URL is correct. I am sure that the URL is correct because I can download it when input the URL in the browser.
